### PR TITLE
Demo PR

### PR DESCRIPTION
**Pull request by dave-vazquez**
# Description

Demo PR

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [x] Unit Test
- [x] Test B

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Comment by dave-vazquez** [comment]
[Onboarding - (FE)](https://trello.com/c/vY45ql5u/30-onboarding-fe)

**Review by CodyBrewer — APPROVED** [review]

**Comment by dillonp23** [comment]
Looks good

File: pages/index.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ import { CenteredContainer } from "../components/Layout/LayoutPrimitives";
 import { useRouter } from "next/router";
 import withApollo from "../lib/apollo";
 
+// demo change
```
Comment (alesslongaretti) [inline pages/index.js:10]: looks good

**Review by alesslongaretti — APPROVED** [review]

**Review by dillonp23 — APPROVED** [review]
Works now needed to refresh

**Pull request merged**